packages/dev/core/src/scene.ts

```diff
@@ -1384,6 +1385,33 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
         return this._texturesEnabled;
     }
 
+    private _useFrameGraph = false;
+    /**
+     * Gets or sets a boolean indicating if the frame graph should be used
+     */
+    public get useFrameGraph(): boolean {
+        return this._useFrameGraph;
+    }
+
+    public set useFrameGraph(value: boolean) {

```

Comment:
ca we simplify it by having the accessor on the frameGraph property? if set we shunt the customRenderFunction and we restore it when framegraph is set to null

Reply:
I'm not sure I understand: you'd like to have `useFrameGraph` in the `FrameGraph` class itself? Wouldn't that be a bit strange? Maybe I should just remove the `Scene.useFrameGraph` property and consider that we should render through the frame graph as soon as the `Scene.frameGraph` property is not null?

Reply:
Yes that's my ask: no need for useFrameGraph